amgut/db/patches/0040.sql

```diff
@@ -0,0 +1,220 @@
+-- Sep. 9, 2016
+-- Plate Mapper test 01
+
+SET search_path TO ag;
+
+-- Create new tables
+
+CREATE TABLE extraction_kit_lot (
+	extraction_kit_lot_id bigserial  NOT NULL,
+	name                 varchar  NOT NULL,
+	description          varchar  ,
+	CONSTRAINT pk_extraction_kit_lot PRIMARY KEY ( extraction_kit_lot_id ),
+	CONSTRAINT uq_extraction_kit_lot_name UNIQUE ( name )
+ );
+
+CREATE TABLE extraction_robot (
+	extraction_robot_id  bigserial  NOT NULL,
+	name                 varchar  NOT NULL,
+	description          varchar  ,
+	CONSTRAINT pk_extraction_robot PRIMARY KEY ( extraction_robot_id ),
+	CONSTRAINT uq_extraction_robot_name UNIQUE ( name )
+ );
+
+CREATE TABLE master_mix_lot (
+	master_mix_lot_id    bigserial  NOT NULL,
+	name                 varchar  NOT NULL,
+	description          varchar  ,
+	CONSTRAINT pk_master_mix_lot PRIMARY KEY ( master_mix_lot_id ),
+	CONSTRAINT uq_master_mix_lot_name UNIQUE ( name )
+ );
+
+CREATE TABLE plate_type (
+	plate_type_id        bigserial  NOT NULL,
+	name                 varchar  NOT NULL,
+	cols                 smallint  NOT NULL,
+	rows                 smallint  NOT NULL,
+	description          varchar  ,
+	CONSTRAINT pk_plate_type PRIMARY KEY ( plate_type_id ),
+	CONSTRAINT uq_plate_type_name UNIQUE ( name )
+ );
+
+CREATE TABLE processing_robot (
+	processing_robot_id  bigserial  NOT NULL,
+	name                 varchar  NOT NULL,
+	description          varchar  ,
+	CONSTRAINT pk_processing_robot PRIMARY KEY ( processing_robot_id ),
+	CONSTRAINT uq_processing_robot_name UNIQUE ( name )
+ );
+
+CREATE TABLE run (
```

Comment:
I intensionally leave the whole run table empty for any fields, because we haven't get a full idea what the lab crew want us to add. Maybe let me talk to them and get the information once and for all?